### PR TITLE
func_frame_trace: Add CLI command to dump frame queue.

### PR DESCRIPTION
**Pull request by InterLinked1**
This adds a simple CLI command that can be used for analyzing all frames currently queued to a channel.

A couple log messages are also adjusted to be more useful in tracing bridging problems.

Resolves: #533

### CODE REVIEW

**Comment by InterLinked1** [comment]
cherry-pick-to: 18
cherry-pick-to: 20
cherry-pick-to: 21

File: main/features.c [hunk 1]
```diff
@@ -1011,7 +1011,7 @@ static int bridge_exec(struct ast_channel *chan, const char *data)
 			strlen(args.dest_chan));
 	}
 	if (!current_dest_chan) {
-		ast_verb(4, "Bridge failed because channel '%s' does not exist\n",
+		ast_log(LOG_WARNING, "Bridge failed because channel '%s' does not exist\n",
```
Comment (jcolp) [inline main/features.c:1014]: This seems like an unrelated logging change

**Review by jcolp — CHANGES_REQUESTED** [review]

File: main/features.c (hunk 1 above)
Comment (InterLinked1) [reply main/features.c:1014]: It's related in that if bridging fails, it's now noticeable to the user. Before it was just a verbose, even though something went wrong.

But I'll put up another review at some point for log message related changes so if you'd rather it go with that, I can move it there.

Comment (jcolp) [reply main/features.c:1014]: That's a separate thing and should be a separate review.

**Review by jcolp — APPROVED** [review]
